react/examples.spec.jsx

```diff
@@ -60,6 +60,7 @@ testComponent('Paper')
 testComponent('PasswordExample')
 testComponent('PercentageBar')
 testComponent('PercentageLine')
+testComponent('PieChart') // Prints error: can't acquire context from the given item
```

Comment:
👀 (I will add a reply soon to this point)

Reply:
=>  `yarn add -D canvas` solved this problem.

Actually, when diving deeper inside chart.js, we can catch this error - and here is the solution: https://github.com/hustcc/jest-canvas-mock/issues/2

Reply:
merci d'avoir creusé ce sujet ! J'ai pris la solution https://github.com/hustcc/jest-canvas-mock/issues/2#issuecomment-500307391 de ton lien